mail_optional_follower_notification/models/mail_thread.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class MailThread(models.AbstractModel):
+    _inherit = "mail.thread"
+
+    def _notify_compute_recipients(self, message, msg_vals):
+        """ Compute recipients to notify based on subtype and followers. This
```

Comment:
this overrides the original docstring - if any - you should use a comment instead.

Reply:
That's the docstring of the overrided method.

Reply:
But then you should:

* mention that it extends the original behavior
* multi line docstring should be separated 

Anyway, there's no official OCA rule for this (yet).
Feel free to ignore my remark :wink: